simapp/v2/app_config.go

```diff
@@ -151,11 +151,11 @@ var (
 					// NOTE: The genutils module must also occur after auth so that it can access the params from auth.
 					InitGenesis: []string{
 						consensustypes.ModuleName,
-						accounts.ModuleName,
 						authtypes.ModuleName,
 						banktypes.ModuleName,
 						distrtypes.ModuleName,
 						stakingtypes.ModuleName,
+						accounts.ModuleName,
```

Comment:
why was this moved?

Reply:
When creating a lockup account in genesis, the accounts must be placed below the bank and staking because parameters above this are referenced.